lib/appearance.rb

```diff
@@ -110,6 +113,10 @@ def inner_word
     #
     # Returns Nokogiri::XML object
     def rss_feed
-      Nokogiri::XML(open(url))
+      xml = Rails.cache.fetch("xml", :expires_in => 1.day) do
```

Comment:
INFO: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)